nap/fields.py

```diff
@@ -153,6 +161,26 @@ def descrub_value(self, val, for_read=False):
         return obj_list
 
 
+class SimpleListField(Field):
+    """A list of simple types (e.g. strings, not Resources)"""
+    def scrub_value(self, val):
+        if isinstance(val, list):
+            return val
+        elif val is None:
+            return []
+        else:
+            return [val]
+
+    def descrub_value(self, val, for_read=False):
+        if isinstance(val, list):
+            return val
+        elif val is None:
+            return []
+        else:
+            return [val]
```

Comment:
Great point. I'll handle this and add a test and then merge.
On Apr 15, 2015 12:06 PM, "David Nelson Adamec" <notifications@github.com>
wrote:

> In nap/fields.py
> <https://github.com/CashStar/nap/pull/7#discussion_r28434707>:
>
> > +    """A list of simple types (e.g. strings, not Resources)"""
> > +    def scrub_value(self, val):
> > +        if isinstance(val, list):
> > +            return val
> > +        elif val is None:
> > +            return []
> > +        else:
> > +            return [val]
> > +
> > +    def descrub_value(self, val, for_read=False):
> > +        if isinstance(val, list):
> > +            return val
> > +        elif val is None:
> > +            return []
> > +        else:
> > +            return [val]
>
> I like the way it's currently written. I think we should change falsey
> objects like '', 0, and False to [''] [0], and [False].
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CashStar/nap/pull/7/files#r28434707>.
>